components/admin_console/reset_email_modal/index.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import ResetEmailModal from './reset_email_modal.jsx';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
```

Comment:
Returning this `... ownProps` is not necessary and I think this entire index.js as well.